### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by karthykarthick**
### The purpose of this pull request is to evaluate the Milestone 2 branch and merge it into the Development branch.
**Main** features:

-  Added the App, ButtonPanel, Button, and Display components.
-  Showing some information by using props.

### CODE REVIEW

File: src/components/app.js
Comment (trekab) [inline src/components/app.js]: Wrap children components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

File: src/index.js
Comment (trekab) [inline src/index.js]: Apart from being a bad practice to add scripts this way, I don't think you need to import big.js this way since you already added this package to the package.json file.

**Review by trekab — CHANGES_REQUESTED** [review]
#### Changes Required! :recycle: 
Hi @karthykarthick! Your effort to implement this milestone's requirements so far is commendable. There are some issues I'd like you to work on before finishing this milestone though. These include:
- [ ] There are three button panels displayed on your app and several(10) `jsx` warnings in the console. Please fix this.
- [ ]   Please provide a descriptive README.md file for your project. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] Check the comments in the code below for more helpful feedback.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: src/components/app.js
Comment (teekaytech) [reply src/components/app.js]: This change, as requested by the last reviewer, has not been implemented. Please do so. You can check out an example of React Fragment from the link above.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this milestone.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Kindly push your recent changes to production (Heroku)
- [ ] You're having two(2) App.js and one of them is redundant. The required `App.js` should be placed in `src/components` directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png)). Kindly remove the other one.
- [ ] Check my inline comments below and make changes as appropriate.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @karthykarthick   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

**1. Just one issue left to fix which is your live link demo.
I'm curious is it working fine at your end 🤔 ?
And also kindly update your README file, you should have `Milestone 2` in it not `milestone 1`**

**_Heroku Link Issue:_** 
milestone-2



### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @karthykarthick congratulations on your head-work :muscle: Having fixed the header in the README file which now references milestone2 and also having fixed the live link which now runs on netlify from the Heroku link previously :smile: 

I hereby approve the project :fireworks: 
Kindly go ahead and merge into development, continue with the next milestone :slightly_smiling_face: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**
